README.md

```diff
@@ -20,6 +20,13 @@ The package can be installed as:
 
 ## Usage
 
+Start Double in your `test/test_helper.exs` file:
+
+```elixir
+ExUnit.start
+Application.ensure_all_started(:double)
```

Comment:
Hmm, I'm not sure. All other libraries I've seen usually don't offer such convenience functions. Actually, if you use Elixir 1.4 and don't declare list of applications in `mix.exs` explicitly, Mix will start `double` application for you.

Like this:
```
def applications do
  [extra_applications: [:logger]]
end

def deps do
  [{:double, ">= 0.0.0", only: :test}]
end
```

But I can add `Double.start/0` if you want